### PR TITLE
update: Removing @emotion

### PR DESCRIPTION
**Pull request by carlagn**

### CODE REVIEW

**Comment by nilubava** [comment]
@carlagn Looks like the package.json needs more reverting in this PR. It should be exactly like the package.json in v0.340.

File: src/website/WebsiteHeader.tsx [hunk 1]
```diff
@@ -1,438 +1,578 @@
...
+                              </DD.Link>
+                              <DD.Link
+                                lightTheme={lightTheme}
+                                href={"https://prisma.io/blog/"}
+                              >
+                                Blog
+                              </DD.Link>
+                              <DD.Link
+                                lightTheme={lightTheme}
+                                href={"https://prisma.io/jobs/"}
```
Comment (nilubava) [inline src/website/WebsiteHeader.tsx:398]: Please change '/jobs' to '/careers'
also, make the same change in footer link for 'Careers'

File: src/website/WebsiteFooter.tsx [hunk 2]
```diff
@@ -124,52 +124,116 @@ const WebsiteFooter = ({ newsletterComponent }: any) => (
...
+          </Link>
+          <Link
+            href={`${
+              absoluteLinks ? "https://www.prisma.io" : ""
+            }/docs/getting-started/`}
+          >
             Get Started
           </Link>
-          <Link href="https://github.com/prisma/prisma-examples">
+          <Link href="https://github.com/prisma/prisma-examples/">
```
Comment (nilubava) [inline src/website/WebsiteFooter.tsx:166]: there is no trailing slash here

File: src/website/WebsiteFooter.tsx [hunk 3]
```diff
@@ -124,52 +124,116 @@ const WebsiteFooter = ({ newsletterComponent }: any) => (
...
+            Data Browser
+          </Link>
+          <Link
+            href={`${
+              absoluteLinks ? "https://www.prisma.io" : ""
+            }/data-platform/proxy/`}
+          >
+            Data Proxy
+          </Link>
+          <Link href="https://www.prisma.io/pricing/">Pricing</Link>
```
Comment (nilubava) [inline src/website/WebsiteFooter.tsx:151]: No trailing slash for any of these links

**Comment by nilubava** [comment]
@carlagn Looks like there is trailing slash for all the links here - Trailing slashes are present only for the following links:
- prisma.io/docs/
- prisma.io/blog/
- prisma.io/dataguide/

Rest all the links should be without trailing slash

File: src/website/WebsiteFooter.tsx [hunk 4]
```diff
@@ -124,52 +124,116 @@ const WebsiteFooter = ({ newsletterComponent }: any) => (
...
+            href={`${absoluteLinks ? "https://www.prisma.io" : ""}/dataguide/`}
+          >
+            Data Guide
+          </Link>
 
-          <Link href="https://www.prisma.io/prisma-in-your-ecosystem">
+          <Link
+            href={`${
+              absoluteLinks ? "https://www.prisma.io" : ""
+            }/prisma-in-your-ecosystem`}
```
Comment (nilubava) [inline src/website/WebsiteFooter.tsx:180]: Please change this to `/stack`

**Review by sampolahtinen — APPROVED** [review]
Tested locally in PDP, by creating a local package out of this branch using yalc.

Results:
- Menu works
- Dialog works
- All e2e tests pass

Good to go from PDP side! :) 

FYI: @tatianainama  @luanvdw 

Thank you so much once again!

**Pull request merged**